packages/@webex/internal-plugin-metrics/src/call-diagnostic/call-diagnostic-metrics-latencies.ts

```diff
@@ -169,6 +169,30 @@ export default class CallDiagnosticLatencies extends WebexPlugin {
     );
   }
 
+  /**
+   * Getting catalog time from U2C service
+   * @returns - latency
+   */
+  public getU2CTime() {
+    const preAuth =
+      this.getDiffBetweenTimestamps(
+        'internal.get.u2c.request-preauth',
+        'internal.get.u2c.response-preauth'
+      ) || 0;
+
+    const postAuth =
+      this.getDiffBetweenTimestamps(
+        'internal.get.u2c.request-postauth',
+        'internal.get.u2c.response-postauth'
+      ) || 0;
+
+    const calculatedLatency = preAuth + postAuth;
```

Comment:
question: If one of the two are missing (maybe there is an in-flight request), the latency will be much lower (approximately half) of what is expected, is this OK?